### PR TITLE
Wrote some documentation on some trig functions, along with code samples.

### PR DESCRIPTION
**Pull request by billymeter**
A few changes in this PR:

- Documentation and code examples for `ABS`, `ATN`, `COS`, `SIN`, `TAN` functions
- Added my name to `AUTHORS` file
- Corrected typo in `README.md`

### CODE REVIEW

File: .github/workflows/makefile.yml [hunk 1]
```diff
@@ -12,7 +12,8 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
-
+    permissions:
+      contents: write
```
Comment (jestin) [inline .github/workflows/makefile.yml:16]: This shouldn't be needed, as I have it at a global level above on lines 3-4.  Please remove so I can merge this PR.

File: README.md
Comment (jestin) [inline README.md:1]: Just like in the LaTeX files, the standard for the Markdown files are using 2 spaces at the end of sentences.  Can you please change back all the places where you've changed them to 1 space?

File: .github/workflows/makefile.yml (hunk 1 above)
Comment (billymeter) [reply .github/workflows/makefile.yml]: Not sure how this got in here. Must have slipped in my local copy when attempting to test the permissions issues we had with the GitHub Action. It's corrected in the revised commit.

**Comment by billymeter** [comment]
Apologies for the whitespace issues. My editor did that automatically on save. I've reverted to the original whitespacing in `README.md`.

**Comment by jestin** [comment]
I cleaned this up and merged it manually.  Closing.

**Pull request closed without merging**